mscore/palette/palettewidget.cpp

```diff
@@ -119,10 +120,13 @@ void PaletteWidget::setupStyle()
 
 void PaletteWidget::activateSearchBox()
       {
+      int delay = isFloating() ? 300 : 0;
```

Comment:
May seem strange, but when palette is floating, the focus in event ends up occuring "after" the focus in event of the searchText and ends up then removing the preparation for search string. Adding a slight delay to the request _only_ when floating clears this up